.github/CODEOWNERS

```
@@ -1,3 +1,3 @@
 # Automatically generated CODEOWNERS
 # Regenerate with `make update-codeowners`
-draft-ietf-oauth-sd-jwt-vc.md oliver.terbu@spruceid.com mail@danielfett.de
+draft-ietf-oauth-sd-jwt-vc.md oliver.terbu@spruceid.com mail@danielfett.de bcampbell@pingidentity.com
```

github says "this CODEOWNERS file contains errors" and "Unknown owner on line 3: make sure a user with the email address mail@danielfett.de exists and has write access to the repository"
  
```suggestion
draft-ietf-oauth-sd-jwt-vc.md oliver.terbu@spruceid.com fett@danielfett.de bcampbell@pingidentity.com
```